govc/session/login.go

```diff
@@ -97,7 +95,7 @@ Examples:
   govc session.login -u host -extension com.vmware.vsan.health -cert rui.crt -key rui.key
   token=$(govc session.login -u host -cert user.crt -key user.key -issue) # HoK token
   bearer=$(govc session.login -u user:pass@host -issue) # Bearer token
-  token=$(govc session.login -u host -cert user.crt -key user.key -issue -actas "$bearer")
+  token=$(govc session.login -u host -cert user.crt -key user.key -issue -token "$bearer")
```

Comment:
Hi @dougm,

I like this change. I think the flag `token` is much more natural and friendly.